parts/linux/cloud-init/artifacts/cse_cmd.sh

```diff
@@ -1,7 +1,10 @@
-echo $(date),$(hostname) > /var/log/azure/cluster-provision-cse-output.log;
-cloud-init status --wait  > /dev/null 2>&1;
-[ $? -ne 0 ] && echo 'cloud-init failed' >> /var/log/azure/cluster-provision-cse-output.log && exit 1;
-echo "cloud-init succeeded" >> /var/log/azure/cluster-provision-cse-output.log;
+PROVISION_OUTPUT="/var/log/azure/cluster-provision-cse-output.log";
+echo $(date),$(hostname) > ${PROVISION_OUTPUT};
+{{if not ShouldDisableCustomData}}
```

Comment:
idea: flip it around to be 

{{ if ShouldEnableCustomData}}

i feel like it will be easier to read without the double negative?

Reply:
I agree, bool names should ideally never contain 'negatives'

Reply:
@alexeldeib what is your opinion on the name "ShouldEnableCustomData" for the function for the sake of readability?
Mikolaj, Cameron and I had a side discussion offline, I would like the variable name to remain DisableCustomData since the crux of the matter here is disabling custom data + I want to make sure it aligns with RP toggle, but I would be open to renaming the function to be better readable with the bash script. 

also welcome other suggestions!

Reply:
changing the function in baker.go and leaving the datamodel sounds fine to me. I agree the bash is wordy

Reply:
done!